src/org/labkey/test/util/Crawler.java

```diff
@@ -477,7 +477,7 @@ else if (isAbsoluteUrl(urlText)) // Make sure it is a link to the site under tes
                 String relativeURL;
                 try
                 {
-                    relativeURL = "/" + WebTestHelper.makeRelativeUrl(urlText);
+                    relativeURL = WebTestHelper.makeRelativeUrl(urlText);
```

Comment:
Whats was the reason to add the "/" in the first place?

Reply:
I added it in the previous PR because I thought tests want it. I was incorrect.